src/seccompiler/Cargo.toml

```diff
@@ -12,25 +12,19 @@ bench = false
 
 [[bin]]
 name = "seccompiler-bin"
-path = "src/seccompiler_bin.rs"
+path = "src/bin.rs"
```

Comment:
If nothing depends on this as a library component anymore, then we should probably this file to `main.rs` and remove the library target from the crate (and maybe rename the entire crate `seccompiler_bin`)

Reply:
The Firecracker build script uses this as a lib.

Reply:
ah right. nvm then. maybe we can deprecate seccompiler-bin altogether, and remove it in 2.0. but that's neither here nor there.